manticore/ethereum/state.py

```diff
@@ -1,6 +1,18 @@
 from ..core.state import StateBase
+from ..core.plugin import StateDescriptor
 
 
 class State(StateBase):
     def execute(self):
+        super().execute()
         return self._platform.execute()
+
+    def _update_state_descriptor(self, descriptor: StateDescriptor, *args, **kwargs):
+        """
+        Called on execution_intermittent to update the descriptor for this state.
+        This one should apply any EVM-specific information to the descriptor.
+
+        :param descriptor: StateDescriptor for this state
+        """
+        super()._update_state_descriptor(descriptor, *args, **kwargs)
+        descriptor.pc = None  # TODO - Grab whatever notion of a PC EVM has
```

Comment:
@feliam & @montyly - What details about the ManticoreEVM state would you be interested in having made available via the introspection API?

Reply:
If I understand what a StateDescriptor is correclty it should show a proyection of what happened in the state?
Instruction count? TX trace length? current PC ?

A PC in EVM would look like: `(state.platform.current_vm.address, state.platform.current_vm.pc)` 
The contract account and the pc inside that. Maybe that's enough for the initial support?

You can print `state.platform.current_vm` that would take a photo of that vm. (slow) ?